README.md

```diff
@@ -77,13 +77,12 @@ The binaries built by this repository can be run with any VMM which
 can act as a vhost-user frontend. Typically they have been tested with
 [QEMU](https://www.qemu.org) although the rust-vmm project does
 provide a [vhost-user
-frontend](https://github.com/rust-vmm/vhost/tree/main/src/vhost_user)
+frontend](https://github.com/rust-vmm/vhost/tree/main/vhost/src/vhost_user)
 crate for rust based VMMs.
 
 While it's possible to implement all parts of the backend inside the
 vhost-device workspace consideration should be given to separating the
-VirtQueue handling and response logic to a crate in [vm-virtio
-devices](https://github.com/rust-vmm/vm-virtio/tree/main/crates/devices).
+VirtQueue handling and response logic to the crate [vm-virtio](https://github.com/rust-vmm/vm-virtio).
```

Comment:
`vm-virtio` is a workspace so not a crate:

```suggestion
VirtQueue handling and response logic to a device crate in the [vm-virtio](https://github.com/rust-vmm/vm-virtio) repository.
```

Reply:
Thank you, @epilys. You were absolutely right, and the issue has now been corrected.